ui/components/ui/sender-to-recipient/sender-to-recipient.component.js

```diff
@@ -8,6 +8,7 @@ import { shortenAddress } from '../../../helpers/utils/util';
 import AccountMismatchWarning from '../account-mismatch-warning/account-mismatch-warning.component';
```

Comment:
The changes in this file account for the new behaviour of clicking the address shown at the top of the confirm screen (and in the transaction details modal):


https://user-images.githubusercontent.com/7499938/144241598-f4b01da6-6820-494d-ac67-8b9e0a04ecb1.mp4